tests/strands/agent/test_agent.py

```diff
@@ -626,7 +624,7 @@ def test_agent_tool_tool_does_not_exist(agent):
 
 
 @pytest.mark.parametrize("tools", [None, [tool_decorated]], indirect=True)
-def test_agent_tool_names(tools, agent):
+def test_agent_tool_names(agent):
```

Comment:
The pytest parametrize attribute still references tools here - is that expected?

Is it still doing what it should be doing in that case?